internal/docker/docker.go

```diff
@@ -453,19 +454,34 @@ func (handler *Handler) ContainerRemove(ctx context.Context, srcContainerID stri
 
 // ProjectDir returns the project directory as is configured for the given image.
 func (handler *Handler) ProjectDir(ctx context.Context, imageID string) (string, error) {
-	ii, _, err := handler.client.ImageInspectWithRaw(ctx, imageID)
+	// The image can tell us via a label where saucectl should mount the project files.
+	// We default to the working dir of the container as the default mounting target.
+	return handler.getObjectLabel(ctx, imageID, "com.saucelabs.project-dir")
+}
+
+// ChromeVersion returns the version of chrome installed in the given imageID
+func (handler *Handler) ChromeVersion(ctx context.Context, imageID string) (string, error) {
+	return handler.getObjectLabel(ctx, imageID, "selenium_chrome_version")
```

Comment:
🔪 selenium 🔪 
I'd rather we use our own label that's consistent with our naming scheme like a la `com.saucelabs.project-dir`.
Otherwise we'll have to continue using `selenium_chrome_version` going forward 😢

Reply:
👍 , I'll add a more compliant label to the testrunner image.

Reply:
`com.saucelabs.*-version` is the format for the labels, e.g. `com.saucelabs.chrome-version`